force-app/main/default/reports/NPSP_Outbound_Fund_Reports/Unscheduled_Disbursements_for_Awarded_Gr.report-meta.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <columns>
+        <field>CUST_NAME</field>
+    </columns>
+    <columns>
+        <field>Funding_Request__c.Awarded_Date__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Funding_Request__c.Awarded_Amount__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Funding_Request__c.Total_Remaining__c</field>
+    </columns>
+    <description>List of funding requests with available amount for disbursement.</description>
+    <filter>
+        <criteriaItems>
+            <column>Funding_Request__c.Total_Remaining__c</column>
+            <columnToColumn>false</columnToColumn>
+            <isUnlocked>true</isUnlocked>
+            <operator>greaterThan</operator>
+            <value>0</value>
+        </criteriaItems>
+        <language>en_US</language>
+    </filter>
+    <format>Tabular</format>
+    <name>Unscheduled Disbursements for Awarded Gr</name>
```

Comment:
@nicolersmith - The name of the report can't be longer than 40 characters. The name requested is `Unscheduled Disbursements for Awarded Grants` but it's too long. We need to shorten the name.